webapi/src/it/scala/org/knora/webapi/responders/admin/GroupsResponderADMSpec.scala

```diff
@@ -238,15 +238,14 @@ class GroupsResponderADMSpec extends CoreSpec with ImplicitSender {
         )
 
         val statusChangeResponse = expectMsgType[GroupOperationResponseADM](timeout)
-
         statusChangeResponse.group.status shouldBe false
 
         appActor ! GroupMembersGetRequestADM(
           groupIri = SharedTestDataADM.imagesReviewerGroup.id,
           requestingUser = SharedTestDataADM.rootUser
         )
 
-        expectMsg(Failure(NotFoundException("No members found.")))
+        expectMsg(Failure(NotFoundException(s"Group <${SharedTestDataADM.imagesReviewerGroup.id}> was deleted.")))
```

Comment:
This change does not correspond to what is described in the issue. I would have expected an empty `List` which should result in a 200 response here and not a `Failure` which will result in a 404 response.

From the issue:
>  If no groups exist, I would expect receiving an empty list, not an error.

Reply:
@seakayone I think this is conflating two things:
1. requesting all groups if there are 0 groups => should return []
2. requesting the members for a group, which is deleted => should return 404 because the group is deleted

Reply:
@seakayone quoting part is fullfilled in `groupsGetRequestADM` method. Line you are reffering to is something additiona I did, which is also in the PR description and even better describet by @BalduinLandolt above.

Reply:
Thanks Balduin for the clarification, from the PR description "Improved error login for deleted groups." it was not apparent to me that this means the 2. case.

However, I would also have expected a change in our tests for the 1. case.